src/server/cli/vdiff/migrate.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+import { appendFile, mkdir, readFile, rename, rm } from 'node:fs/promises';
+import { join, normalize, parse } from 'node:path';
+import commandLineArgs from 'command-line-args';
+import { glob } from 'glob';
+import { PATHS } from '../../visual-diff-plugin.js';
+import { stdout } from 'node:process';
+
+export async function migrate(argv = []) {
+	const { pattern = './**' } = commandLineArgs({ name: 'pattern', type: String, defaultOption: true }, { partial: true, argv });
```

Comment:
Only changes to this from the `bin` version were wrapping it in a function and changing the default pattern from `./test/**` to `./**` because there's no reason to restrict where to look for goldens by default.

Reply:
Will this look in `node_modules`?  I remember that coming up with the old library

Reply:
It will, but it's being ignored on line 12